scripts/components/PaymentForm.jsx

```diff
@@ -0,0 +1,128 @@
+var React = require('react');
+var moment = require('moment');
+var Api = require('../utils/Api.js');
+var MuiContextified = require('./MuiContextified.jsx');
+var mui = require('material-ui');
+var DatePicker = mui.DatePicker;
+var TextField = mui.TextField;
+var RaisedButton = mui.RaisedButton;
+var Paper = mui.Paper;
+
+var PaymentForm = React.createClass({
+  getInitialState() {
+    return {
+      fullName: '', // user entered full name on credit card
+      cardNumber: '', // user entered credit card number
+      expiryDate: '', // user entered expiry date
+      ccv: '', // user entered ccv number
+      amount: '', // user entered amount
+    }
+  },
+  propTypes: {
+    paymentAdded: React.PropTypes.func,
+  },
+
+  // Capture the input field state after each keypress.
+  onChange(field, event) {
+    this.setState({ [field]: event.target.value });
+  },
+
+  // Handle the form submission event when the user adds new repair request.
+  onSubmit(event) {
+    event.preventDefault();
+    event.stopPropagation();
+
+    // API call to add payment
+    Api.addPayment({
+      data: {
+        // fullName: this.state.fullName,
+        // cardNumber: this.state.cardNumber,
+        // expiryDate: this.state.expiryDate,
+        // ccv: this.state.ccv,
+        amount: this.state.amount
+      },
+      callback: (err, response) => {
+        if (err) {
+          return;
+        }
+        // Clear the form
+        this.setState({
+          fullName: "",
```

Comment:
Can you follow the style of the codebase. Strings in JavaScript should be single quoted. Eg. `fullName: ''`